docs/workloads/azure/backend/java_cqrs/requirements_java_cqrs.md

```diff
@@ -4,6 +4,17 @@ title: Requirements
 sidebar_label: Requirements
 hide_title: false
 hide_table_of_contents: true
+description: Requirements for get started with the Java Rest API application
```

Comment:
Better as `Requirements for getting started with the Java Rest API application`

Reply:
fixed